lib/digital_collex/indexer.ex

```diff
@@ -0,0 +1,31 @@
+defmodule DigitalCollex.Indexer do
+  alias DigitalCollex.Indexer
+
+  def get_figgy_catalog(getter \\ Indexer) do
+    apply(getter, :get_url, ["https://figgy.princeton.edu/catalog.json?rows=100"])
```

Comment:
Can we try `getter.get_url("https://figgy.princeton.edu/catalog.json?rows=100")`?